pkg/cli/image/append/append.go

```diff
@@ -57,8 +57,10 @@ var (
 		add the --drop-history flag to remove information from the image about the system that
 		built the base image.
 
-		Images in manifest list format will automatically select an image that matches the current
-		operating system and architecture unless you use --filter-by-os to select a different image.
+		Images in manifest list format with keep-manifest-list specified will automatically append layers
+		to all sub manifests in the list unless filter-by-os is specified in which case the append will
+		only happen for the filtered manifests while preserving the manifestlist. If keep-manifest-list is
+		not specified, a single manifest is filtered and appended to depending on the filter.
```

Comment:
Do we want to keep the information about the automatic selection of an image when keep-manifest-list is not used? i.e, this part from the previous doc: 
```
automatically select an image that matches the operating system and architecture
```